build/vagrant-dependencies.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+sudo add-apt-repository universe
+sudo apt update
+sudo apt install software-properties-common
```

Comment:
```suggestion
sudo apt-get update
sudo apt-get install software-properties-common
```

When run from a script, apt complains about being unsuitable for scripts. Would it be an idea to change this to use `apt-get` instead? (Would necessitate a change to the `depenedencies.sh` as well to be consistent).

Reply:
I can try it, unsure if it will work the same. I'll see!

Reply:
Looks like it still works with apt-get 👍